slidez-fe-core/src/containers/poll-editor/PollEditor.tsx

```diff
@@ -73,19 +119,24 @@ const PollEditor: React.FC<PollEditorProps> = ({ presentationId }) => {
                                                     key={index}
                                                     className='options'
                                                 >
-                                                    <Field
-                                                        name={`options.${index}.title`}
-                                                        placeholder='Your option'
-                                                        className='input-options'
-                                                    />
+                                                    <div className='label option-label'>
+                                                        {`Option ${index + 1}`}
+                                                        <Field
+                                                            name={`options.${index}.title`}
+                                                            placeholder='Your option'
+                                                            className='input'
+                                                        />
+                                                    </div>
                                                     <button
                                                         type='button'
                                                         className='options-btn'
-                                                        onClick={() =>
-                                                            arrayHelpers.remove(
-                                                                index
-                                                            )
-                                                        }
+                                                        onClick={() => {
+                                                            if (index > 0) {
+                                                                arrayHelpers.remove(
+                                                                    index
+                                                                )
+                                                            }
```

Comment:
Move it to function
const toggleRemoveClick = () => { if (index > 0) ....}

Reply:
Done